src/webgpu/web_platform/copyImageBitmapToTexture.spec.ts

```diff
@@ -175,25 +175,25 @@ got [${failedByteActualValues.join(', ')}]`;
 export const g = makeTestGroup(F);
 
 g.test('from_ImageData')
-  .params(
+  .cases(
+    poptions('dstColorFormat', [
+      'rgba8unorm',
+      'bgra8unorm',
+      'rgba8unorm-srgb',
+      'bgra8unorm-srgb',
+      'rgb10a2unorm',
+      'rgba16float',
+      'rgba32float',
+      'rg8unorm',
+      'rg16float',
+    ] as const)
+  )
+  .subcases(() =>
     params()
       .combine(poptions('width', [1, 2, 4, 15, 255, 256]))
       .combine(poptions('height', [1, 2, 4, 15, 255, 256]))
       .combine(poptions('alpha', ['none', 'premultiply']))
```

Comment:
Maybe the alpha and orientation could be cases too? They feel like something that would be easy to be only partially implementing.